src/hooks/screen/useCreateChat.ts

```diff
@@ -93,17 +124,20 @@ const useCreateChat = () => {
     }
   };
 
-  const handleAnonymousMessage = async (selectedUser) => {
+  const handleAnonymousMessage = async (selectedUser, channelId = null, context = null) => {
     if (!selectedUser?.allow_anon_dm) {
       SimpleToast.show('This user does not allow anonymous messages');
       return;
     }
 
     try {
       const response = await getOrCreateAnonymousChannel(
-        selectedUser?.user_id || selectedUser.userId
+        selectedUser?.user_id || selectedUser.userId,
+        channelId,
+        context
       );
-      const chatData = createChannelJson(response, selectedUser);
+
+      const chatData = createChannelJson(response, selectedUser, true);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `handleAnonymousMessage` function now includes `channelId` and `context` parameters, which are passed to the `getOrCreateAnonymousChannel` function. This change looks good as it allows more flexibility when creating or retrieving an anonymous channel. However, please ensure that all calls to `handleAnonymousMessage` have been updated to include these new parameters where necessary.


<!-- This is an auto-generated comment by OSS CodeRabbit -->